src/config/syscheck-config.c

```diff
@@ -1258,7 +1258,7 @@ int read_data_unit(const char *content) {
     if (content[len_value_str - 1] == 'B' || content[len_value_str - 1] == 'b') {
         if (isalpha(content[len_value_str - 2])){
             os_calloc(len_value_str, sizeof(char), value_str);
-            strncpy(value_str, content, len_value_str - 2);
+            memcpy(value_str, content, len_value_str - 2);
```

Comment:
Same as on some other instance, wouldn't using `memcpy` not copy the null terminator?

Reply:
Unless we don't actually need the null terminator..

Reply:
Hi @RamiroRD 

The `os_calloc` function uses the `calloc` function internally, because of this we always initialize all elements to 0 and we will have a null-terminated string.

Best regards
Hanes